ui/app/components/secret-create-or-update.js

```diff
@@ -40,7 +38,7 @@ const LIST_ROOT_ROUTE = 'vault.cluster.secrets.backend.list-root';
 const SHOW_ROUTE = 'vault.cluster.secrets.backend.show';
 
 export default class SecretCreateOrUpdate extends Component {
-  @tracked codemirrorString = null;
+  // @tracked codemirrorString = null;
```

Comment:
Did you mean to keep this as a comment?

Reply:
I don't think so. Guessing I was trying to debug something so let me uncomment it and see if the tests still pass.

Reply:
So commenting that out did [fix these tests](https://app.circleci.com/pipelines/github/hashicorp/vault/43011/workflows/ed15d699-f40f-4258-ad2a-66613495c3d1/jobs/526190) which are now failing. I must have forgot to loop back and look for a proper solution. I'm thinking the code-mirror modifier must be send the on change event right away which is triggering the same computation error.